packages/cf-component-box/README.md

```diff
@@ -0,0 +1,53 @@
+# cf-component-box
+
+> Cloudflare Box Component
+
+This is styling component similar to cf-component-flex which allows for arbitrary styling.
+
+This component should only be used when the built in style doesn't quite fit the bill and when
+it can't be done with fela. Ideally, cf-ui components should have the right styling built in for
+most cases. In the few edge cases that pop up in real world feature development, this is an alternative
+to wrapping your component in a `<div>` or `<span>` and styling it with css to accomodate one-offs.
```

Comment:
@koddsson please don't use inline styles anymore and try out fela instead, it's exactly like inline styles but you can use everything that you know from CSS (media queries, ::before ...) and it's actually using `<style></style>` so the performance is much better